std/assembly/index.d.ts

```diff
@@ -149,6 +149,27 @@ declare function fmod(x: f64, y: f64): f64;
 /** Returns the 32-bit floating-point remainder of `x/y`. */
 declare function fmodf(x: f32, y: f32): f32;
 
+declare namespace Atomic { 
```

Comment:
What about use `Atomics` instead `Atomic` but for only for namespace with generic methods? The same as provided [Web API](https://developer.mozilla.org/ru/docs/Web/JavaScript/Reference/Global_Objects/Atomics#%D0%9C%D0%B5%D1%82%D0%BE%D0%B4%D1%8B)? This also takes effect to `Atomics.exchange` and `Atomics.compareExchange`.